andela/ninjas-ah-frontend
# #165412880 Article pagination

#### What does this PR do?

#### Description of Task to be completed?
- Article Pagination
- When an article read time is less than a minute, `read time` displayed is `1 min read`


#### How should this be manually tested?
After cloning the repo, run the command `npm run dev`, and then check at the bottom of articles list

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#165412880 

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/10960862/62003060-f8023c80-b110-11e9-99df-7084c0957c02.png)

#### Questions:
